examples/counter/vendors/vendor.js

```diff
@@ -47,3 +47,13 @@
   }
   window['FrintReact'] = m;
 })();
+
+// frint-store
+(function () {
```

Comment:
Sure but following what we discussed yesterday I will do it for the kitchensink.